frontend/package.json

```diff
@@ -3,12 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
+    "@fortawesome/free-brands-svg-icons": "^5.14.0",
+    "@fortawesome/free-regular-svg-icons": "^5.14.0",
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.11",
```

Comment:
Eftersom Andreas har designat med Feather Icons (https://github.com/feathericons/react-feather), så snacka vi i Teams om att använda det istället. Du får gärna flika in med din egen mening. :relaxed: